src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -1551,4 +1561,65 @@ public static boolean isReciprocalOverlap(final SimpleInterval interval1, final
                 (interval1.intersect(interval2).size() >= (interval2.size() * reciprocalOverlapThreshold)) &&
                 (interval2.intersect(interval1).size() >= (interval1.size() * reciprocalOverlapThreshold));
     }
+
```

Comment:
I think it makes sense to push these methods down.

Is it necessary to allow the header to be optional?  I think the format of each row (strand and target name) is specific to a Picard interval_list.

Other than that, I'm not going to review this code too closely.